libs/sdk-ui-geo/src/index.ts

```diff
@@ -13,7 +13,10 @@
 export {
     IGeoConfig,
     IGeoLngLat,
+    IGeoPushpinChartBaseProps,
+    GeoPushpinChartPropsUnion,
```

Comment:
Should we export this type?

Reply:
we need to export it as it is used by other exported entities, eg. typeguards

Reply:
I have removed also these typeguards, as they are needed only internally too